docker-bake.hcl

```diff
@@ -0,0 +1,434 @@
+# TODO: Add the lint checks for the following conditions:
+# - All container images really are accounted for in the "all" group
+# - No group includes any target that starts with "_"
+# - Targets that start with "_" can only appear in `inherits` lists
+#   - Should *only* targets that begin with "_" appear in `inherits` lists?
+# - All the targets in the "cloudsmith-images" group are configured
+#   with the `upstream_aware_tag` function
+# - All the targets *not* in the "cloudsmith-images" group are *not*
+#   configured with the `upstream_aware_tag` function
+# - Introspect all targets to find all context directories and ensure
+#   that a .dockerignore file exists in each location
+
+# Variables
+########################################################################
+# Variables in this section are intended to be set by users to
+# influence the build.
+
+# IMAGE_TAG is the "master variable"; if it is set to something other
+# than "latest", we will build images suitable for production
+# usage. If it is unset, or "latest", we'll be creating images for
```

Comment:
Can you update this with to mention that dev will create images for local production. ie  `if it is unset, "dev", or "latest"...`
You have dev mentioned in the comments below, but since this is the variable that users should be updating they're more likely to read this comment